backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -489,7 +489,7 @@ private NonRootCreatedAndSkippedBranches createGitMacheteNonRootBranch(
       String branchName = coreLocalBranch.getName();
 
       // Section spans the characters before the first dot
-      // Name spans the characters after the first dot
+      // Name spans the characters after the last dot
```

Comment:
While you're at this file... line 874, `s/distinct/distinguish` (adjective -> verb)